plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlConfig.java

```diff
@@ -62,7 +62,8 @@ public boolean isEnableStringPushdownWithCollate()
         return enableStringPushdownWithCollate;
     }
 
-    @Config("postgresql.experimental.enable-string-pushdown-with-collate")
+    @Config("postgresql.enable-string-pushdown-with-collate")
```

Comment:
Does it require [docs](https://trino.io/docs/current/connector/postgresql.html#predicate-pushdown-support) update in this PR or in a follow-up?